src/com/projectkorra/projectkorra/Element.java

```diff
@@ -40,23 +40,23 @@ public String getBend() {
 
 	private static final HashMap<String, Element> ALL_ELEMENTS = new HashMap<>(); // Must be initialized first
 
-	public static final Element AIR = new Element("Air");
-	public static final Element WATER = new Element("Water");
-	public static final Element EARTH = new Element("Earth");
-	public static final Element FIRE = new Element("Fire");
-	public static final Element CHI = new Element("Chi", ElementType.BLOCKING);
-	public static final Element AVATAR = new Element("Avatar", null);
-	public static final SubElement FLIGHT = new SubElement("Flight", AIR, ElementType.NO_SUFFIX);
-	public static final SubElement SPIRITUAL = new SubElement("Spiritual", AIR, ElementType.NO_SUFFIX);
-	public static final SubElement BLOOD = new SubElement("Blood", WATER);
-	public static final SubElement HEALING = new SubElement("Healing", WATER, ElementType.NO_SUFFIX);
-	public static final SubElement ICE = new SubElement("Ice", WATER);
-	public static final SubElement PLANT = new SubElement("Plant", WATER);
-	public static final SubElement LAVA = new SubElement("Lava", EARTH);
-	public static final SubElement METAL = new SubElement("Metal", EARTH);
-	public static final SubElement SAND = new SubElement("Sand", EARTH);
-	public static final SubElement LIGHTNING = new SubElement("Lightning", FIRE);
-	public static final SubElement COMBUSTION = new SubElement("Combustion", FIRE);
+	public static final Element AIR = new Element("Air", "GRAY", "[Air]");

```

Comment:
Again, good idea but these values could easily be pulled from the language file allowing for users to still customize our chat formatting.
```java
public static final Element AIR = new Element("Air", ConfigManager.languageConfig.get().getString("Chat.Colors.Air"), ConfigManager.languageConfig.get().getString("Chat.Prefixes.Air"));
```